src/IO/H5/SpectralIo.hpp

```diff
@@ -17,12 +17,12 @@ namespace h5_detail {
 /// We maintain a list of bases and quadratures which are compatible
 /// with IO, this allows for efficient storage of Spectral::Basis in
 /// volume data files (currently there are 3 bases)
-std::array<Spectral::Basis, 3> allowed_bases();
+std::array<Spectral::Basis, 4> allowed_bases();
```

Comment:
Update comment above (delete the outdated part of the comment I think, it's redundant).

Same for quadratures below.